mobile/src/test/java/org/openhab/habdroid/model/ParsedStateTest.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.habdroid.model
+
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class ParsedStateTest {
+    @Test
+    fun testParseAsBoolean() {
+        listOf("ON", "100", "1", "0.1").forEach {
```

Comment:
0.1 is detected as brightness and converted to int -> becomes 0.

This commit only contains the new failing test.